### ISSUE TITLE
Define number of votes to approve a string

### ISSUE THREAD

**Issue by nukeador** [Common-Voice/sentence-collector#4]
- Currently if a sentences has 1 positive review is approved
- Currently if a sentences has 1 negative review is rejected

We should see if we want to implement at least 2-3 votes to approve and also keep the sentences with negative votes at the end of the list just in case.

We want to avoid a single user to make the final call for sentences and avoid abuse.

**Comment by mikehenrty** [comment]
Good point. Should we do the same as Common Voice and just use the 2 out of 3 rule? @nukeador

**Comment by nukeador** [comment]
@mikehenrty which rule are we using for voice reviews?

**Comment by mikehenrty** [comment]
2 out of 3 rule: if 2 people out of 3 think a clip is valid or invalid, we add it to that bucket

**Comment by nukeador** [comment]
Let's follow that for now but get that configurable somewhere, we might need to move it lower if we don't have enough reviews.

**Comment by nukeador** [comment]
After a conversation with @MichaelKohler last week, we think the review process can lead to a high drop in contributions if a lot of people have to review once sentence but since this is just our assumption, we want to validate it with this tool.

We will follow the current way of operating: You need at least one other person to review a sentence to be validated.

The number of people needed to validate a sentence will be configurable so in case we see a lot of poor quality sentences we can increase it.
